Makefile.core.mk

```diff
@@ -21,7 +21,7 @@ BAZEL_TARGETS ?= //...
 # Don't build Debian packages and Docker images in tests.
 BAZEL_TEST_TARGETS ?= ${BAZEL_TARGETS}
 E2E_TEST_TARGETS ?= $$(go list ./...)
-E2E_TEST_FLAGS ?=
+E2E_TEST_FLAGS ?= -p=1 -parallel=1
```

Comment:
Golang log muxing is absolutely atrocious. Making it single threaded so we can actually parse the logs.